composer.json

```diff
@@ -29,7 +29,7 @@
     },
     "require-dev": {
         "mockery/mockery": "^0.9",
-        "behat/behat": "^3.0.15",
+        "behat/behat": "^3.0",
```

Comment:
This doesn't change anything. Behat 3.1 was already supported by this constraint.

See https://getcomposer.org/doc/articles/versions.md#caret

The test is breaking for a different reason